source/simulator/parameters.cc

```diff
@@ -302,6 +302,42 @@ namespace aspect
                        "the composition system gets solved. See 'linear solver "
                        "tolerance' for more details.");
 
+
+    prm.enter_subsection("Formulation");
+    {
+      prm.declare_entry ("Formulation", "custom",
+                         Patterns::Selection ("isothermal compression|custom|ALA|BA"),
+                         "Select a formulation for the basic equations. Different "
+                         "published formulations are available in ASPECT (see list of "
+                         "possible parameters for available options). Two ASPECT specific "
+                         "options are: 'isothermal compression': ASPECT's original formulation, "
```

Comment:
maybe make this into an enumerated list? This way you avoid the two colons in one line.

Use `\`` as an opening backtick for latex.